maldoca/base/get_runfiles_dir.cc

```diff
@@ -29,9 +29,10 @@
 #error "__FreeBSD__ not supported."
 #endif
 
-#if defined(PLATFORM_WINDOWS)
-// TODO(#110) Windows-specific headers.
-#error "PLATFORM_WINDOWS not supported."
+#if defined(_WIN32)
```

Comment:
not important now but we  should stick with one style of macro:  ifdef vs if defined()

Reply:
ack